project/src/js/ui/tournament/detail/tournamentDetailView.js

```diff
@@ -94,6 +139,21 @@ function variantInfo(data) {
   return variant;
 }
 
+function variantKey(data) {
```

Comment:
Ok, I wasn't seeing a `perf` key in the tournament object. Will look again and if not there, I'll ask @ornicar if he can add.